crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -48,6 +48,9 @@ pub const BRP_REPARENT_METHOD: &str = "bevy/reparent";
 /// The method path for a `bevy/list` request.
 pub const BRP_LIST_METHOD: &str = "bevy/list";
 
+/// The method path for a `bevy/reparent` request.
+pub const BRP_MUTATE_COMPONENT_METHOD: &str = "bevy/mutate_component";
```

Comment:
I'm not sure what the convention is for multiple words in a method name is. Should the method be named differently? I'm trying to consider future methods for modifying other things like Resource's. 

I had a few other ideas for methods:

- `bevy/modify/component`
- `bevy/mutate/component`
- `bevy/component_mut` *this feels too Rusty in this context*